_inc/client/components/settings/index.jsx

```diff
@@ -29,7 +29,7 @@ export const Settings = React.createClass( {
 					activated={ this.props.isSettingActivated( window.Initial_State.settingNames.jetpack_holiday_snow_enabled ) }
 					toggleSetting={ this.props.toggleSetting }
 					disabled={ this.props.isFetchingSettingsList }
-				>Show falling snow on my blog until January 4<sup>th</sup>.</SettingToggle>
+				>{ __( 'Show falling snow on my blog until January 4th.' ) }</SettingToggle>
```

Comment:
@oskosk maybe we can add now what @richardmuscat pointed in pb6Nl-b1o-p2 in comment 83464
> We either remove it or change the text to “show falling snow on my blog from XX-XX-XXX to XX-XX-XXXX”
so it would be 
> Show falling snow on my blog from Dec 1st until Jan 4th.